inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -361,34 +313,6 @@ def prizes2marc(self, key, value):
     }
 
 
-@hepnames.over('_public_note', '^680..')
-@utils.for_each_value
-def _public_note(self, key, value):
```

Comment:
There are actually public notes. We should add it back to the schema. Actually this is yet a meta-field that is valuable in all schemas.